nucleus/parent/pom.xml

```diff
@@ -88,7 +88,7 @@
         <!-- Jakarta SOAP (with Attachments)
        <jakarta.xml.soap-api.version>3.0.0-RC2</jakarta.xml.soap-api.version>
         -->
-        <webservices.version>4.0.3</webservices.version>
+        <webservices.version>4.0.4</webservices.version>
```

Comment:
hmm, looking at other deps, it will probably fail again unless EL-API gets updated to 6.0.0 and Servlet-API to 6.1.0; XML-WS related APIs should be updated to their latest versions too, but should not cause any troubles if not given their updates are fully backward compatible

Reply:
We can't update those in GF 7 to those versions, since they are Jakarta EE 11 dependencies.

GlassFish 8 is for Jakarta EE 11, GlassFish 7 is for Jakarta EE 10.

What we really need is a Metro / WebServices with Jakarta EE 10 dependencies (so no Expression Language 6 and no Servlet 6.1), but with WoodStox 7.

Reply:
@lukasj Can you do a Metro / Webservices 4.0.4 with only WoodStox updated to 7 (or, even better, just widen the protection in OSGi to include 6 and 7). Then do a Metro / Webservices 4.1.x for Jakarta EE 11?

Reply:
No. Try 4.0.4 again. It should be ok for use in EE 10 as well as 11 now. Given EE 10 requires SE 11, I see no point in keeping woodstox 6.x allowed.

anyway _XML-WS related APIs should be updated to their latest versions_ note remains valid; `<jakarta.xml.soap-api.version>3.0.0-RC2</jakarta.xml.soap-api.version>` is definitely wrong even for EE 10 final

Reply:
The 3.0.0-RC2 only seems to occur in a comment:

```xml
   <!-- Jakarta SOAP (with Attachments)
       <jakarta.xml.soap-api.version>3.0.0-RC2</jakarta.xml.soap-api.version>
        -->
```

Reply:
This is in server.log of the clustered instance. I have no idea why the test passed locally. Some cached dependency on CI from staging repo ?
```
Caused by: java.lang.ClassNotFoundException: jakarta.servlet.ServletContainerInitializer not found by org.glassfish.metro.webservices-osgi [299]
	at org.apache.felix.framework.BundleWiringImpl.findClassOrResourceByDelegation(BundleWiringImpl.java:1591)
	at org.apache.felix.framework.BundleWiringImpl.access$300(BundleWiringImpl.java:79)
	at org.apache.felix.framework.BundleWiringImpl$BundleClassLoader.loadClass(BundleWiringImpl.java:1976)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:525)
	... 96 more
]]
```

Reply:
org.glassfish.* is being cached on the CI indeed. If Metro uses that and Lukas re-staged 4.0.4, that might be the issue.

There is a job to clear the cache.

Reply:
@lukasj can we release metro / webservices 4.0.4 to Maven Central?